applications/dot_marketplace-Phase3.md

```diff
@@ -0,0 +1,222 @@
+| --- | --- | --- |
+| 0a | License | Apache 2.0 |
+| 0b | Documentation | We will provide both inline documentation of the code and a tutorial that explains how a user can use DOT Marketplace and understand the flow of tasking pallet. |
+| 0c | Testing Guide | Functions will be covered by unit tests, the documentation will describe how to run these tests. We will also provide scripts to help deploy, run and test the build. |
+| 0d | Docker Image | Docker image of the build |
+| 1 | Project Structure | The existing application only allows one milestone per project. Phase 3 modifies it to allow a publisher to add multiple milestones under the same project. |
+| 2 | Multiple Bidders| Multiple bidders can now bid for the same milestone, and the publisher can choose one worker based on the bidder ratings |
+| 3 | Escrow  | Multiple subaccounts are created for a project to account for each milestone and make it easier to store all funds for transfer/exchange. |
+
+### **Milestone 2**
+
+* **Estimated duration:** 2.0 weeks
+* **FTE:** 1
+* **PTE:** 2
+* **Costs:** 9,225 USD
+
+
+In continuation to previous work, this milestone involves the creation of an on-chain decentralized court to handle dispute resolution. Each milestone can go into a dispute on the same scope as mentioned in the second phase of dot marketplace. The other milestones in a project are not affected by the dispute of one of the milestones. The court pallet will be upgraded to support these new features.
+
+| Sr no. | Deliverable | Description |
+| --- | --- | --- |
+| 0a | License | Apache 2.0 |
+| 0b | Documentation | We will provide both inline documentation of the code and a tutorial that explains how a user can use DOT Marketplace and understand the flow of tasking pallet. |
+| 0c | Testing Guide | Functions will be covered by unit tests, the documentation will describe how to run these tests. We will also provide scripts to help deploy, run and test the build. |
+| 0d | Docker Image | Docker image of the build |
+| 1 | Decentralized Court Module | An on-chain decentralized court for dispute resolution within the ecosystem. |
+| 1a | Disapprove Milestone  | In the case of a customer not being satisfied with the work submitted by the service provider (worker). A set of jurors is shortlisted (court summon) to resolve the dispute and pass a verdict. |
+| 1b | Disapprove Rating | The customer or the service provider, once they have received their rating for a particular milestone and are not satisfied with it. |
+| 1c | General Dispute | A general dispute function for cases that do not fall under the categories mentioned in 1a and 1b. |
+| 2 | Voting module | Each juror can review the dispute and cast their vote, which also includes their rating for both the customer and the worker. After two days, all the juror votes are counted, and a winner is identified. |
+| 3 | Frontend App  | Supporting frontend UI to test the aforementioned functionality. |
```

Comment:
Do you have wireframes to demonstrate what this approximately would look like?

Reply:
We will be providing a skeletal UI to showcase the features of dot_marketplace. Similar, to the Phase1 submission.

Reply:
[Phase1 Guide](https://github.com/WowLabz/dot_marketplace_docker/tree/Phase1_Milestone3]) has the UI screenshots for the same. This will be updated for the Phase3 functionalies.